main/consolekit2/APKBUILD

```diff
@@ -1,24 +1,28 @@
-# Contributor: Bart Ribbers <bart.ribbers@openmailbox.org>
-# Maintainer: Bart Ribbers <bart.ribbers@openmailbox.org>
+# Contributor: Bart Ribbers <bribbers@disroot.org>
+# Maintainer: Bart Ribbers <bribbers@disroot.org>
 pkgbase=ConsoleKit2
 pkgname=consolekit2
 pkgver=1.2.0
-pkgrel=1
-pkgdesc="A framework for defining and tracking users, login sessions, and seats. "
+pkgrel=2
+pkgdesc="A framework for defining and tracking users, login sessions, and seats"
+provides=consolekit
+replaces=consolekit
 arch=all
 url="https://consolekit2.github.io/ConsoleKit2"
 license=GPL2
-depends="gettext glib zlib polkit eudev libdrm cgmanager libnih"
+depends="gettext glib zlib polkit eudev libdrm libnih"
```

Comment:
I don't think gettext, glib, zlib, libdrm and libnih needs to be in depends. They should be automatically pulled in.